src/app/test_executive/zkapps.ml

```diff
@@ -624,9 +676,13 @@ module Make (Inputs : Intf.Test.Inputs_intf) = struct
       section_hard "Wait for zkApp transaction to mint token"
         (wait_for_zkapp parties_mint_token)
     in
+    let%bind () =
+      section_hard "Wait for zkApp transaction to mint 2nd token"
+        (wait_for_zkapp parties_mint_token2)
+    in
     let%bind () =
       section_hard "Wait for zkApp transaction to transfer tokens"
-        (wait_for_zkapp parties_token_transfer)
+        (long_wait_for_zkapp parties_token_transfer)
```

Comment:
I think it is because of the bulk payments. Each block can only take 2 user commands and we send a few payments to produce snarked ledger that tries to get in first. You could maybe move the padding payments (sent using `send_padding_transactions`) to the end and see if parties_token_transfer gets included sooner?

Or you could increase the transaction fee of parties_token_transfer so that it is larger than payments' fee but smaller/equal to the fee of mint token parties

Reply:
running the experiment of moving the padding payments to the end, locally ...